SageMaker/README.md

```diff
@@ -0,0 +1,46 @@
+# Run the Hugging Face notebooks on SageMaker 
+
```

Comment:
Maybe a short intro here: E.g. "Here are the instructions to run the book's notebooks on AWS SageMaker."